src/main/java/net/imglib2/mesh/Meshes.java

```diff
@@ -375,4 +376,58 @@ public static void scale( final Mesh mesh, final double[] scale )
 			vertices.set( i, x * scale[ 0 ], y * scale[ 1 ], z * scale[ 2 ] );
 		}
 	}
+
+	public static BufferMesh merge( final Iterable< Mesh > meshes )
```

Comment:
I would suggest to accept a `Mesh out` as argument, and concatenate them all into that, rather than forcing the result to be `BufferMesh`.